pkg/profiler/cpu/cpu.go

```diff
@@ -77,54 +78,65 @@ const (
 	configKey                = "unwinder_config"
 )
 
-type Config struct {
+// UnwinderConfig gets sync to BPF module.
+type UnwinderConfig struct {
 	FilterProcesses   bool
 	VerboseLogging    bool
 	MixedStackWalking bool
+	Python            bool
+	Ruby              bool
+}
+
+type Config struct {
+	ProfilingDuration          time.Duration
+	ProfilingSamplingFrequency uint64
+
+	PerfEventBufferPollInterval       time.Duration
+	PerfEventBufferProcessingInterval time.Duration
+	PerfEventBufferWorkerCount        int
+
+	MemlockRlimit uint64
+
+	DebugProcessNames []string
+
+	DWARFUnwindingDisabled         bool
+	DWARFUnwindingMixedModeEnabled bool
+	BPFVerboseLoggingEnabled       bool
+
+	PythonUnwindingEnabled bool
+	RubyUnwindingEnabled   bool
+}
```

Comment:
Thanks for separating the configs, it's way nicer now and long overdue to be honest 😄